.github/workflows/code-quality.yml

```diff
@@ -32,6 +32,20 @@ jobs:
         run: mvn -B -T1C -DskipTests -DskipChecks install
       - name: Perform CodeQL Analysis
         uses: github/codeql-action/analyze@v2
+  spotbug:
+    name: SpotBugs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      - name: Set up JDK 17
+        uses: actions/setup-java@v3.3.0
+        with:
+          distribution: 'temurin'
+          java-version: '17'
+          cache: 'maven'
+      - name: Spot bugs
+        run: mvn -B -T1C -DskipTests verify -Pspotbugs
```

Comment:
:wrench: We could skip all checks except SpotBugs :man_shrugging:

Reply:
How would you do that? `-DskipChecks` and then?

Reply:
It was just a suggestion :smile: Anyway, yes, `skipChecks` doesn't skip SpotBugs right now since by default without the profile SpotBugs is skipped. So you can safely add `skipChecks`. If you wanted to be safe, you could also add `-Dspotbugs.skip=false` just in case somebody would set it to true ;)

Reply:
:+1: I'll do that together with the sarif reports: https://github.com/camunda/zeebe/issues/9311#issuecomment-1119619378